src/securejoin.rs

```diff
@@ -326,6 +326,14 @@ pub(crate) async fn handle_securejoin_handshake(
                 ChatId::create_for_contact(context, contact_id).await?;
             }
 
+            // TODO Here we need to check that the token isn't too old (no more than 2 days)
```

Comment:
Why do we need this check at this step? Just requesting the auth data is safe anyway. Looks like doing the check before `chat::add_contact_to_chat_ex()` is sufficient